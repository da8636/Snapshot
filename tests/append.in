SET a 1 2
GET a
APPEND a 3 4
GET a
BYE
