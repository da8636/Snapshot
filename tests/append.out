ok

[1 2]

ok

[1 2 3 4]

bye
